docs_src/general/ContainerNames.md

```diff
@@ -35,17 +35,19 @@ The following table provides the list of the default EdgeX Docker image names to
     |edgexfoundry/device-grove|edgex-device-grove|edgex-device-grove|device-grove|
     |edgexfoundry/device-coap|edgex-device-coap|edgex-device-coap|device-coap|
 === "Security"
-    |Docker image name|Docker container name |Docker network hostname|Docker Compose service name|
+    |Docker image name|Docker container name |Docker network hostname|Docker Compose service name|Note|
     |---|---|---|---|
     |vault|edgex-vault|edgex-vault|vault|
-    |postgress|edgex-kong-db|edgex-kong-db|kong-db|
-    |kong|edgex-kong|edgex-kong|kong|
+    |nginx|edgex-nginx|edgex-nginx|nginx|
+    |postgress|edgex-kong-db|edgex-kong-db|kong-db|DEPRECATED|
+    |kong|edgex-kong|edgex-kong|kong|DEPRECATED|
```

Comment:
We are beyond deprecation as @farshidtz points out.
```suggestion

```

Reply:
Removed altogether.